ui/src/views/compute/MigrateWizard.vue

```diff
@@ -168,6 +170,12 @@ export default {
         this.hosts.sort((a, b) => {
           return b.suitableformigration - a.suitableformigration
         })
+        for (const key in this.hosts) {
+          if (this.hosts[key].suitableformigration && !this.hosts[key].requiresstoragemigration) {
+            this.hosts.unshift({ id: -1, name: 'autoselect', suitableformigration: true, requiresstoragemigration: false })
```

Comment:
@weizhouapache do we need to add this string - `autoselect` in translation?

Reply:
@shwstppr done.